rigex-front-end/src/components/authInput/AuthInput.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import {
+  Input,
+  IconButton,
+  FormControl,
+  InputLabel,
+  Grid,
+} from '@material-ui/core';
+import {
+  Mail,
+  Lock,
+  VisibilityOutlined,
+  VisibilityOffOutlined,
+} from '@material-ui/icons';
+
+import './authInput.scss';
+
+const AuthInput = ({
```

Comment:
I think you'd find this more re-usable if you had different input components that kept their own logic for their input type.

EmailInput, PasswordInput, etc. and just wrapped them in the form control where necessary.

These components would be easier to test and use individually, and you could re-use them elsewhere without worrying about them being in a form with a certain name.

For example, you might want another place to "send reset password email" that uses the EmailInput, and then a reset password screen that has 2 PasswordInputs for confirmation of the new password.

I'm not sure of the roadmap of this project but you never know.

Reply:
Broke out email and password into two different components